pages/common/source.md

```diff
@@ -0,0 +1,7 @@
+# source
+
+> Evaluate contents of file.
+
+- Cause fish to re-read its initialization file:
```

Comment:
We need an example which shows a basic usage. Something like source {{file.sh}} would do.

Reply:
How you would change the example of mine: `source ~/.config/fish/config.fish`?

Reply:
This isn't a `fish`-specific command - `bash` has it too. Perhaps we could adapt the description here a bit?

Reply:
@sbrl I have added two more commands. And changed description too.

Reply:
I still see `Cause fish to re-read its initialization file` here. Was it a different description you changed?

Reply:
Sounds good to me! A generic `{{path/to/file}}` example is all that's needed I think.

Reply:
Maybe `Evaluates contents of file` or `Execute commands from a file` (same as main description)?

Reply:
`Evalute contents of a given file` sounds good.